ovs/extensions/healthcheck/testing/decorators.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2016 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+import time
+import threading
+import unittest
+from ovs.extensions.healthcheck.decorators import ensure_single_with_callback_cluster
+from ovs.extensions.healthcheck.decorators import ensure_single_with_callback_local
+
+
+class ClusterCheckTester(unittest.TestCase):
+
+    def setUp(self):
+        self.concurreny_amount = 5
```

Comment:
This looks like actual test logic inside the `setUp`. Wouldn't it make more sense to move this into the test itself?